src/services/TechRecordsService.ts

```diff
@@ -334,6 +336,30 @@ class TechRecordsService {
     const notifiableAlterationIds = ["47", "48"];
     return notifiableAlterationIds.includes(testTypeId);
   }
+
+  public async updateEuVehicleCategory(systemNumber: string, newEuVehicleCategory: EU_VEHICLE_CATEGORY): Promise<HTTPResponse | HTTPError> {
+    const techRecordWrapper: ITechRecordWrapper = (await this.getTechRecordsList(systemNumber, STATUS.ALL, SEARCHCRITERIA.SYSTEM_NUMBER))[0];
+    const nonArchivedTechRecord = techRecordWrapper.techRecord.filter((techRecord) => techRecord.statusCode !== STATUS.ARCHIVED);
+    if (nonArchivedTechRecord.length > 1) {
+        throw new HTTPError(400, HTTPRESPONSE.EU_VEHICLE_CATEGORY_MORE_THAN_ONE_TECH_RECORD);
+    }
+    if(nonArchivedTechRecord[0].euVehicleCategory) {
+        return new HTTPResponse(200,HTTPRESPONSE.NO_EU_VEHCILE_CATEGORY_UPDATE_REQUIRED);
```

Comment:
typo in NO_EU_**VEHCILE**_CATEGORY_UPDATE_REQUIRED

Reply:
fixed